app/controllers/attachments_controller.rb

```diff
@@ -14,7 +14,7 @@ def index
   def create
     uploaded_file = params.fetch('attachment_file', params.fetch('files', []).first)
```

Comment:
Should we take this PR and clean up both controllers? E.g. why `'files'` instead of `:files`?